snuba/utils/streams/topics.py

```diff
@@ -45,6 +45,7 @@ class Topic(Enum):
     PROFILE_CHUNKS = "snuba-profile-chunks"
 
     REPLAYEVENTS = "ingest-replay-events"
+    UPTIME_MONITOR_CHECKS = "snuba-uptime-monitor-checks"
```

Comment:
change this to uptime results

Reply:
need to actually figure out if we need to enrich stuff from postgres, or we modify uptime checker